src/main/java/net/testudobank/MvcController.java

```diff
@@ -395,10 +404,18 @@ public String submitDispute(@ModelAttribute("user") User user) {
       // If reversing a deposit puts customer back in overdraft
       if (reversalAmountInPennies > userBalanceInPennies){
         // check if the reversed deposit helped pay off overdraft balance
```

Comment:
Add to this comment saying something like "if it did, do not re-apply the interest rate after the reversal of the deposit since the customer was already in overdraft."